Classes/Console/Composer/InstallerScript/PopulateCommandConfiguration.php

```diff
@@ -51,8 +51,7 @@ public function run(ScriptEvent $event): bool
             /** @var PackageInterface $package */
             $installPath = ($installPath ?: $basePath);
             if (in_array($package->getType(), ['metapackage', 'typo3-cms-extension', 'typo3-cms-framework'], true)) {
-                // Commands in TYPO3 extensions are scanned for anyway at a later point.
-                // With that we ensure not showing commands for extensions that aren't active.
+                // Commands in TYPO3 extensions are scanned any more. They should rather use DI configuration to register commands.
```

Comment:
should it be called "are **not** scanned anymore"?

Reply:
yes